src/internal/connector/exchange/service_iterators.go

```diff
@@ -75,28 +76,34 @@ func filterContainersAndFillCollections(
 		cID := ptr.Val(c.GetId())
 		delete(tombstones, cID)
 
-		currPath, locPath, ok := includeContainer(qp, c, scope)
-		// Only create a collection if the path matches the scope.
-		if !ok {
-			continue
-		}
-
 		var (
 			dp          = dps[cID]
 			prevDelta   = dp.delta
-			prevPathStr = dp.path
+			prevPathStr = dp.path // do not log: pii
 			prevPath    path.Path
+			ictx        = clues.Add(
+				ctx,
+				"container_id", cID,
+				"previous_delta", prevDelta)
```

Comment:
is the delta string going to have PII in it? Seems like it would have user display name or ID at least?

Reply:
Ah, check, thanks.